tests/product/base_product_case.py

```diff
@@ -36,7 +36,7 @@
 from tests.product.topology_installer import TopologyInstaller
 from tests.product.yarn_slider.slider_installer import SliderInstaller
 
-PRESTO_VERSION = r'presto-main:.*'
+PRESTO_VERSION = r'.*'
```

Comment:
Maybe change it to `r'.+'` so that at least one character is required. Otherwise, the current regex will match an empty string.